src/generators/static/tsConfigFileGenerator.ts

```diff
@@ -81,9 +83,11 @@ export function generateTsConfig(project: Project) {
     highLevelTsConfig.include.push("samples-dev/**/*.ts");
     highLevelTsConfig.compilerOptions["paths"] = {};
     highLevelTsConfig.compilerOptions["paths"][clientPackageName] = ["./src/index"];
-    restLevelTsConfig.include.push("samples-dev/**/*.ts");
-    restLevelTsConfig.compilerOptions["paths"] = {};
-    restLevelTsConfig.compilerOptions["paths"][clientPackageName] = ["./src/index"];
+    if (hasRLCSamplesGenerated) {
```

Comment:
why do we need one more condition for this ?

Reply:
To ensure we do have samples to generated. Sometimes we enable sample option but don't have any sample to be generated. In HLC sample gen we use the condition directly `clientDetails?.samples?.length && clientDetails?.samples?.length > 0`